modules/matrix_operations/include/matrix_operations.h

```diff
@@ -0,0 +1,58 @@
+// Copyright 2021 Kiseleva Anastasia
+
+#ifndef MODULES_MATRIX_OPERATIONS_INCLUDE_MATRIX_OPERATIONS_H_
+#define MODULES_MATRIX_OPERATIONS_INCLUDE_MATRIX_OPERATIONS_H_
+
+#include<vector>
+#include<cmath>
+
+class Matrix {
+ private:
+    unsigned int rows;
+
+    unsigned int cols;
+
+    std::vector<std::vector<double>> data;
+
+
+ public:
```

Comment:
```suggestion
    std::vector<std::vector<double>> data;


 public:
```
Будет достаточно одной пустой строки перед спецификатором public, выглядит неаккуратно с 3мя.